src/libs/sqlite/sqlite-lib.pri

```diff
@@ -8,7 +8,8 @@ INCLUDEPATH += $$PWD
 
 unix:!bsd: LIBS += -ldl
 
-include(../3rdparty/sqlite/sqlite.pri)
+!os2:include(../3rdparty/sqlite/sqlite.pri)
+os2: LIBS += -lsqlite3
```

Comment:
Isn't there a Qt Creator configure option for that? Anyway, if there is none, here's the better version:
```suggestion
os2: LIBS += -lsqlite3
else:include(../3rdparty/sqlite/sqlite.pri)
```

Reply:
I don't see one.